src/middlewared/middlewared/plugins/virt/volume.py

```diff
@@ -185,8 +185,12 @@ async def import_iso(self, job, data):
         }
 
         def read_input_stream():
-            for stream in job.pipes.input.r:
-                yield stream
+            while True:
+                chunk = job.pipes.input.r.read(normalize_size('1 MIB'))
```

Comment:
We should just pass 1048576 to read() instead of calling normalize_size()

Reply:
Can you just specify the `buffering` argument when opening pipe instead?